app/src/main/java/ch/hikemate/app/ui/components/ElevationGraph.kt

```diff
@@ -14,7 +23,7 @@ import kotlin.math.ceil
  *
  * @param elevationData The list of elevation data
  * @param maxNumberOfPoints The maximum number of points to display (default to 40)
- * @param color The color of the graph (default to black)
+ * @param strokeColor The color of the graph (default to black)
  * @param strokeWidth The width of the graph (default to 3f)
  * @param modifier The modifier for the graph (default to fillMaxSize)
  */
```

Comment:
Warning: You miss documentation for some parameters here

Reply:
Good catch thanks!